account-kit/react/src/components/auth/card/footer/poweredby.tsx

```diff
@@ -1,5 +1,5 @@
-import { AlchemyLogo } from "../icons/alchemy.js";
-import { ls } from "../strings.js";
+import { AlchemyLogo } from "../../../../icons/alchemy.js";
+import { ls } from "../../../../strings.js";
```

Comment:
This was because we refactored the `<PoweredBy />` component to live in a newly created `Footer` folder. It should ease discoverability of the component.